zwave_js_server/model/node.py

```diff
@@ -66,9 +69,13 @@ def __init__(self, client: "Client", data: NodeDataType) -> None:
         super().__init__()
         self.client = client
         self.data = data
-        self.values = {
-            get_value_id(self, val): Value(self, val) for val in data["values"]
-        }
+        self.values = {}
```

Comment:
Try this:
```suggestion
        self.values: Dict[str, Union[ConfigurationValue, Value]] = {}
```

Reply:
Mapping resulted in the following:
```
zwave_js_server/model/node.py:386: error: Unsupported target for indexed assignment ("Mapping[str, Union[Value, ConfigurationValue]]")
zwave_js_server/model/node.py:393: error: Unsupported target for indexed assignment ("Mapping[str, Union[Value, ConfigurationValue]]")
zwave_js_server/model/node.py:408: error: "Mapping[str, Union[Value, ConfigurationValue]]" has no attribute "pop"
```

but switching to Dict eliminated the errors. Forgot that I needed to type `self.values`